### PR TITLE
10.0/zabbix: increase ram on nodes

### PR DESCRIPTION
**Pull request by naumvd95**

### CODE REVIEW

File: servers/partner-ci/10.0/zabbix.yaml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ---
+- parameter:
+    name: plugin-specific-parameters
+    parameters:
+      - text:
+          name: SPECIFIC_PROPS
+          default: |
+            ADMIN_NODE_MEMORY=4096;
```
Comment (rkhozinov) [inline servers/partner-ci/10.0/zabbix.yaml]: set 8192 for admin_node

**Review by rkhozinov — CHANGES_REQUESTED** [review]

File: servers/partner-ci/10.0/zabbix.yaml (hunk 1 above)
Comment (naumvd95) [reply servers/partner-ci/10.0/zabbix.yaml]: at first need to check it  on ci

**Review by rkhozinov — APPROVED** [review]

**Review by rkhozinov — CHANGES_REQUESTED** [review]
remove memory assignment

**Comment by naumvd95** [comment]
seems its not necessary now

**Pull request closed without merging**